### DISCUSSION TITLE
Stripe integration does not work

### DISCUSSION THREAD

**Discussion by AmeerHajAli** [vercel/community#5964] [Bug Report]
### Page to Investigate

stripe template

### Steps to Reproduce

I am evaluating Vercel and I am having difficulties using one of the templates, namely the stripe template. Issues:
P1: Every time it creates a GitHub repo, if I refresh the page, or the second step fails or go to other page, I have to start over and it never recognizes that the repo was already created in step 1 so I have to go and delete it in github and repeat step 1 ... annoying but mitigable. 
P0: The supabase/stripe integration template does not seem to work. I get a message "Completing Installation...
This window will close automatically." and it loads forever. I went to the supabase website and validated that the project was created and it is in "green status" so l am not sure.

Thanks for taking the time to review my issue.

**Comment by leerob** [comment]
Could you share the specific template you're having trouble with please?

**Reply by AmeerHajAli** [reply]
Stripe Subscription Starter--subscription-starter
